docs/pages/framework/config.md

```diff
@@ -61,6 +61,43 @@ default = []
 You can refer to the profile in [Scarb documentation](https://docs.swmansion.com/scarb/docs/guides/defining-custom-profiles.html) for more information.
 
 
+## `build-external-contracts`
+
+The `build-external-contracts` field in the `[target.starknet-contract]` section is crucial for projects relying on external libraries or contracts.
+
+## Purpose
+By default, Scarb does not build contracts that belong to external libraries unless explicitly defined in this field. Missing these definitions might not cause compilation errors but will lead to runtime issues when:
+- The deployed `world` system queries information about missing models.
+- `Torii` interacts with the blockchain and cannot find the required contracts.
+
+## Configuration
+
+To ensure all external contracts are built, update the `build-external-contracts` field like this:
+
+```toml
+[[target.starknet-contract]]
+build-external-contracts = [
+    "dojo::world::world_contract::world",
+    "armory::m_Flatbow"
+]
+```
+For every `#[dojo::model]` defined in your project, the corresponding contract name follows the pattern:
+```text
+m_<ModelName>
+````
+
+## Best Practices
+1. Always include all required contracts in `build-external-contracts` to prevent runtime errors.
+2. Test your configuration locally by running `scarb build` and verifying the generated artifacts.
+3. Validate interactions with external models to ensure they function as expected.
```

Comment:
Add something about the tests, "validate interactions" = writing integration tests.